include/AudioEngineProfiler.h

```diff
@@ -53,11 +55,36 @@ class AudioEngineProfiler
 
 	void setOutputFile( const QString& outputFile );
 
+	enum class DetailType {
+		NoteSetup,
+		Instruments,
+		Effects,
+		Mixing,
+		Count
+	};
+
+	constexpr static auto DetailCount = static_cast<std::size_t>(DetailType::Count);
+
+	void startDetail(const DetailType type) { m_detailTimer[static_cast<std::size_t>(type)].reset(); }
+	void finishDetail(const DetailType type)
+	{
+		m_detailTime[static_cast<int>(type)] = m_detailTimer[static_cast<std::size_t>(type)].elapsed();
```

Comment:
Another `int` that could be `std::size_t`:
```suggestion
		m_detailTime[static_cast<std::size_t>(type)] = m_detailTimer[static_cast<std::size_t>(type)].elapsed();
```